auth-service/services/UserService.go

```diff
@@ -20,7 +20,7 @@ func GetAllUsers(c *fiber.Ctx) error {
 	return c.Status(200).JSON(fiber.Map{"data": users, "total": len(users)})
```

Comment:
users := {
// Todo ...
}

can we change like this in go? 
-> fiber.Map {users}